twitter_ads/http.py

```diff
@@ -141,6 +141,8 @@ def __init__(self, code, headers, **kwargs):
             raw_response_body = self._raw_body
 
         try:
+            if not isinstance(raw_response_body, str):
```

Comment:
Just thinking out loud, if it's a `str` only when `zlib.decompress` gets called (a few lines above this), we should maybe move the `decode()` up in that conditional.